hazelcast/src/hazelcast/client/spi/LifecycleService.cpp

```diff
@@ -89,8 +97,12 @@ namespace hazelcast {
                 util::ILogger &logger = util::ILogger::getLogger();
                 switch (lifecycleEvent.getState()) {
                     case LifecycleEvent::STARTING :
-                        logger.info("LifecycleService::LifecycleEvent STARTING");
+                    {
+                        char msg[100];
+                        util::snprintf(msg, 100, "(%s:%s) LifecycleService::LifecycleEvent STARTING", HAZELCAST_GIT_COMMIT_DATE, HAZELCAST_GIT_COMMIT_ID);
```

Comment:
I did not think it as a strong requirement. And adding code just for that seemed unnecessary to me. That is why I leave it that way.